docs/en/tutorials/how_to.md

```diff
@@ -0,0 +1,60 @@
+# Tutorial 11: Collection of common usage
+
+This tutorial mainly collects common usages in MMDetection
+
+## Backbone network using MMClassification
+
+The Hierarchy Registry function proposed in MMCV allows modules in various algorithm libraries of OpenMMLab to call each other. Users can use backbone networks from MMClassification in MMDetection without implementing a network that already exists in MMClassification for MMDetection as well.
```

Comment:
Users can use backbone networks from MMClassification in MMDetection without implementing a network that already exists in MMClassification for MMDetection as well. -> Users can use backbone networks from MMClassification in MMDetection without implementing a network that already exists in MMClassification.